packages/chains/src/chainMetadata.ts

```diff
@@ -161,4 +162,12 @@ export const chainMetadata: {
     corePoolComptrollerContractAddress: '0x272795dd6c5355CF25765F36043F34014454Eb5b',
     nativeToken: ethToken,
   },
+  [ChainId.UNICHAIN_SEPOLIA]: {
+    name: 'Unichain Sepolia',
+    logoSrc: unichainLogo,
+    explorerUrl: 'https://sepolia.uniscan.xyz',
```

Comment:
Let's use `https://unichain-sepolia.blockscout.com` instead. The contracts were verified in that explorer. We can change to the etherscan-based explorer after the mainnet launch, if we are able to verify the contracts there

Reply:
Let's use uniscan.xyz, now that is available :)